packages/api/test/__fixtures__/sdk/optional-payload/index.ts

```diff
@@ -0,0 +1,148 @@
+import Oas from 'oas';
```

Comment:
This test fixture is to explicitly test how we're handling operations that have required metadata and optional payloads and to make sure that we add the necessary type overloads for them.